test/functional/configuration/configuration.spec.js

```diff
@@ -10,180 +10,76 @@ OF ANY KIND, either express or implied. See the License for the specific languag
-    productionEnvironment: environmentFields("production"),
-    stagingEnvironment: environmentFields("staging"),
-    developmentEnvironment: environmentFields("development"),
-    orgIdField: spectrum.textfield("orgIdField"),
-    orgIdRestoreButton: spectrum.button("orgIdRestoreButton"),
-    edgeDomainField: spectrum.textfield("edgeDomainField"),
-    edgeDomainRestoreButton: spectrum.button("edgeDomainRestoreButton"),
-    edgeBasePathField: spectrum.textfield("edgeBasePathField"),
-    edgeBasePathRestoreButton: spectrum.button("edgeBasePathRestoreButton"),
-    defaultConsent: {
-      inRadio: spectrum.radio("defaultConsentInRadio"),
-      outRadio: spectrum.radio("defaultConsentOutRadio"),
-      pendingRadio: spectrum.radio("defaultConsentPendingRadio"),
-      dataElementRadio: spectrum.radio("defaultConsentDataElementRadio"),
-      dataElementField: spectrum.textfield("defaultConsentDataElementField")
-    },
-    idMigrationEnabled: spectrum.checkbox("idMigrationEnabledField"),
-    thirdPartyCookiesEnabled: spectrum.checkbox(
-      "thirdPartyCookiesEnabledField"
-    ),
-    // Due to limitations of the sandbox where tests are run,
-    // testing prehiding style viewing/editing is limited.
-    prehidingStyleEditorButton: spectrum.button("prehidingStyleEditorButton"),
-    clickCollectionEnabledField: spectrum.checkbox(
-      "clickCollectionEnabledField"
-    ),
-    downloadLinkQualifierField: spectrum.textfield(
-      "downloadLinkQualifierField"
-    ),
-    downloadLinkQualifierRestoreButton: spectrum.button(
-      "downloadLinkQualifierRestoreButton"
-    ),
-    downloadLinkQualifierTestButton: spectrum.button(
-      "downloadLinkQualifierTestButton"
-    ),
-    onBeforeEventSendEditorButton: spectrum.button(
-      "onBeforeEventSendEditorButton"
-    ),
-    contextGranularity: {
-      allField: spectrum.radio("contextGranularityAllField"),
-      specificField: spectrum.radio("contextGranularitySpecificField")
-    },
-    specificContext: {
-      webField: spectrum.checkbox("contextWebField"),
-      deviceField: spectrum.checkbox("contextDeviceField"),
-      environmentField: spectrum.checkbox("contextEnvironmentField"),
-      placeContextField: spectrum.checkbox("contextPlaceContextField")
-    },
-    deleteButton: spectrum.button("deleteInstanceButton")
-  });
-}
+import {
+  addInstanceButton,
+  instancesTabs,
+  resourceUsageDialog,
+  instances
+} from "./helpers/viewSelectors";
 
 createFixture({
   title: "Extension Configuration View",
-  viewPath: "configuration/configuration.html"
+  viewPath: "configuration/configuration.html",
+  requiresAdobeIOIntegration: true
 });
 
 const defaultEdgeDomain = "edge.adobedc.net";
 const defaultEdgeBasePath = "ee";
 const defaultDownloadLinkQualifier =
   "\\.(exe|zip|wav|mp3|mov|mpg|avi|wmv|pdf|doc|docx|xls|xlsx|ppt|pptx)$";
 
-const defaultInitInfo = {
-  company: {
-    orgId: "ABC123@AdobeOrg"
-  },
-  tokens: {
-    imsAccess: "Badtoken"
-  }
-};
```

Comment:
I moved this default init info into `extensionViewController`, which simplified some things.